CHANGELOG.md

```diff
@@ -1,32 +0,0 @@
-# Changelog
```

Comment:
doesn't look like `yarn changelog` was successful 😛

Reply:
Ok, I don't know why this happens, but the `yarn changelog` command doesn't work for me atm... doesn't show any errors and it isn't working on master either...

Reply:
Weird, works here. Might be a Windows issue, gonna look into this later at your computer.